cost/aws/elb/clb_unused/README.md

```diff
@@ -0,0 +1,50 @@
+## AWS Unused Classic Load Balancers (CLB) 
+ 
+### What it does
+This policy checks all CLB to determine if any are unused (have no healthy instances) and allows them to be deleted by the user after approval.
```

Comment:
I think we should spell out Classic Load Balancers here and add the abbreviation like in the title heading.
`Classic Load Balancers (CLB)`

Reply:
Updated as per the above comments. Thanks.